install/helm/gloo/generate/values.go

```diff
@@ -309,7 +309,7 @@ type GatewayProxy struct {
 	Kind                           *GatewayProxyKind            `json:"kind,omitempty" desc:"value to determine how the gateway proxy is deployed"`
 	PodTemplate                    *GatewayProxyPodTemplate     `json:"podTemplate,omitempty"`
 	ConfigMap                      *ConfigMap                   `json:"configMap,omitempty"`
-	CustomStaticLayer              interface{}                  `json:"customStaticLayer,omitempty" desc:"static layer configuration (global overrides for envoy behavior) defined in envoy bootstrap yaml"`
+	CustomStaticLayer              interface{}                  `json:"customStaticLayer,omitempty" desc:"static layer configuration (global overrides for envoy behavior) defined in envoy bootstrap yaml https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/runtime#config-runtime cannot set overload or upstream layers"`
```

Comment:
I know the other descriptions haven't been updated yet, but wondering if we could use sentence-case here since we're editing it anyways?
```suggestion
	CustomStaticLayer              interface{}                  `json:"customStaticLayer,omitempty" desc:"Configure the static layer for global overrides to Envoy behavior, as defined in the Envoy bootstrap YAML. You cannot use this setting to set overload or upstream layers. For more info, see the Envoy docs. https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/runtime#config-runtime"`
```

Reply:
Definitely, I will keep that in mind for the future!
Also I forgot there is the accept changes button so i pushed a thing sorry about that. 

Long story short: yep we can and now have made the change.